src/styles/dimensions.styles.ts

```diff
@@ -0,0 +1,5 @@
+import { Dimensions } from "react-native";
+
+export const screenHeight = Dimensions.get("window").height;
+export const screenWidth = Dimensions.get("window").width;
```

Comment:
I see `Dimensions.get("window")` is used in other parts of the code instead of these constants. Please either remove these constants or replace those instances of `Dimensions.get("window")` with these constants.

Reply:
Fixed

Reply:
Fixed